lib/improver/nbhood/nbhood.py

```diff
@@ -223,17 +185,15 @@ def process(self, cube, mask_cube=None):
         cubes_real = []
         for cube_realization in slices_over_realization:
             if self.lead_times is None:
-                radius = self._find_radii(num_ens)
+                radius = self._find_radii(cube_lead_times=None)
```

Comment:
This call is not necessary anymore. If lead_times is None, then the radii will not be adjusted by the ```_find_radii``` function. As such you can just call:

```
cube_new = self.neighbourhood_method.run(cube_realization, self.radii, mask_cube=mask_cube)
```

Reply:
Changed as requested.